docs/adr/0057-force-request-body-variable-with-processor.md

```
@@ -0,0 +1,131 @@
+proposal and has not yet been confirmed by a second maintainer, and the open
+questions below are unresolved.
+
+## Technical Discussion
+
+> "Is my assumption correct to say we would just allocate double max body limit by
+> setting the variable if it isn't valid JSON/XML? Any chance we can do opt out? I
+> would say we should provide a way to the user to decide whether invalid Jason
+> should be populated or not (length can be populated tho). Either opt in or opt
+> out should be possible."
+> — @jcchavezs ([comment](https://github.com/corazawaf/coraza/issues/1685#issuecomment-5392976005))
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)